keep-ui/entities/workflows/model/yaml.schema.ts

```diff
@@ -66,6 +66,20 @@ const auth0LogsProvider: ProviderMetadataForValidation = {
   query_params: ["log_type", "previous_users"],
 };
 
+const fluxcdProvider: ProviderMetadataForValidation = {
+  type: "fluxcd",
+  config: {
+    kubeconfig: {
+      required: false,
+      description: "Kubernetes config file content",
+      sensitive: true,
+      default: null,
+    },
+  },
+  can_query: true,
+  can_notify: false,
+  query_params: ["kubeconfig", "namespace"],
+};
```

Comment:
hey @Kunal-Darekar! this hack is only for providers with dots in names, like `github.stars`, for fluxcd provider, the schema will be automatically generated in the runtime for validation.

```suggestion
```